### PR TITLE
Add endpoint for likes

### PR DESCRIPTION
**Pull request by HankG**

### CODE REVIEW

File: v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import com.codingfeline.twitter4kt.core.apiUrl
+import com.codingfeline.twitter4kt.core.util.Twitter4ktInternalAPI
+import com.codingfeline.twitter4kt.core.util.appendNotNulls
+import com.codingfeline.twitter4kt.v1.api.getInternalListResponse
+import com.codingfeline.twitter4kt.v1.model.status.Tweet
+import io.ktor.http.Parameters
+
+/**
+ * Get's a non-protected user's timeline of their likes (previously favorites). This can be requested either using their Twitter ID
+ * or by using their screenname. One must be passed in for the call to succeed. This will return the full 280 characters
```
Comment (yshrsmz) [inline v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt:29]: Not passing userId/screenName results in authenticated user's likes, so we should cover this case.

File: v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * @param maxId Returns results with an ID less than (that is, older than) or equal to the specified ID.
+ * @param includeEntities The entities node will not be included when set to false.
+ */
+@OptIn(Twitter4ktInternalAPI::class)
+public suspend fun StatusesApi.likes(
+    userId: Long? = null,
+    screenName: String? = null,
+    count: Int? = null,
+    sinceId: Long? = null,
+    maxId: Long? = null,
```
Comment (yshrsmz) [inline v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt:47]: IDs should be string here

File: v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt (hunk 1 above)
Comment (HankG) [reply v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt:29]: That's interesting I didn't realize that. Yep!

**Comment by HankG** [comment]
I believe the latest commit resolves the requested issues.

**Review by yshrsmz — APPROVED** [review]
Thanks a lot!

I've made a few changes:
- move likes to `favorites` package, as I'd like to keep endpoint and package the same, as much as possible.
- change the name of the method to `list`, from `likes`. Same reason as above.

If you don't like it, please file an issue :pray:

**Pull request merged**